Gemfile.rails42

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'jruby-openssl', :platforms => :jruby
+
+group :test, :remote_test do
+  # gateway-specific dependencies, keeping these gems out of the gemspec
+  gem 'braintree', '>= 2.0.0'
+  gem 'oauth', '0.4.7'
+end
```

Comment:
Should we set these as development dependencies instead? 

(Side note: the oauth gem is outdated and generates a lot of warnings.)